### PR TITLE
Remove logger

### PR DESCRIPTION
**Pull request by lepiolet**
The named pipe was causing problems when restarting dnsmasq, dnsmasq was hanging because the pipe was broken. Avoid using it.

### CODE REVIEW

**Review by danijeel — APPROVED** [review]

File: assets/sdp-dnsmasq [hunk 1]
```diff
@@ -23,10 +22,8 @@ EOF
 mk_dnsmasq_conf
 
 # Start dnsmasq now so it has not pid 1
-rm -f $SDP_LOGS_PIPE ; mkfifo $SDP_LOGS_PIPE
-/sdp-dnsmasq/sdp-logger $SDP_LOGS_PIPE &
 /usr/sbin/dnsmasq --pid-file=/sdp-dnsmasq/dnsmasq.pid --port=53 --local-service \
-		  --listen-address=127.0.0.1 --log-queries -8 $SDP_LOGS_PIPE
+                  --listen-address=127.0.0.1 --log-queries -8 /proc/1/fd/1
```
Comment (thomascellerier) [inline assets/sdp-dnsmasq:26]: Can you use the full arg name `--log-facility`, `-8` sounds like magic.

File: assets/sdp-dnsmasq-set-dns [hunk 2]
```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env sh
 
+[ "${FLOCKER}" != "$0" ] && exec env FLOCKER="$0" flock -x "$0" "$0" "$@" || :
```
Comment (thomascellerier) [inline assets/sdp-dnsmasq-set-dns:3]: Can you add a comment to explain what this is doing.

File: assets/sdp-dnsmasq-set-dns [hunk 3]
```diff
@@ -97,27 +102,32 @@ done
...
 if [[ ! -z $pid ]]; then
     kill -9 $pid
 fi
-/sdp-dnsmasq/sdp-logger $SDP_LOGS_PIPE &
 res=1
 while [[ $res != 0 ]]; do
+    decho "Restarting dnsmasq ..."
     /usr/sbin/dnsmasq --pid-file=/sdp-dnsmasq/dnsmasq.pid --port=53 --local-service \
-		      --listen-address=127.0.0.1 --log-queries -8 $SDP_LOGS_PIPE
+                      --listen-address=127.0.0.1 --log-queries -8 /proc/1/fd/1
```
Comment (thomascellerier) [inline assets/sdp-dnsmasq-set-dns:126]: Can you use the full arg name `--log-facility`, `-8` sounds like magic.

**Review by thomascellerier — APPROVED** [review]
Looks good, just a few comments to explain what is going on would be good.

**Pull request merged**